js/plugins/cloud-sql-pg/src/index.ts

```diff
@@ -84,6 +87,14 @@ export function postgres<EmbedderCustomOptions extends z.ZodTypeAny>(
     tableName: string,
     embedder: EmbedderArgument<EmbedderCustomOptions>;
     embedderOptions?: z.infer<EmbedderCustomOptions>;
+    engine: PostgresEngine;
+    schemaName?: string;
+    contentColumn?: string;
+    embeddingColumn?: string;
+    metadataColumns?: string[];
+    idColumn?: string;
+    metadataJsonColumn?: string;
+    distanceStrategy?: 'cosine' | 'ip' | 'l2';
```

Comment:
We should also support ignoreMetadataColumns. In the configure method we should add the logic to check the inputs and that columns exist like https://github.com/averikitsch/langchainjs/blob/6d78313ae24d020ffc69be3d4bf8d637eaa7f237/libs/langchain-google-cloud-sql-pg/src/vectorStore.ts#L114-L165